src/index.js

```
@@ -1 +1,27 @@
 'use strict';
+
+import 'dotenv/config';
+import express from 'express';
+import cors from 'cors';
+import { authRouter } from './routers/auth.router.js';
+import { errorMiddleware } from './middlewares/errorMiddleware.js';
+import cookieParser from 'cookie-parser';
+
+const PORT = process.env.PORT || 8080;
+
+const app = express();
+
+app.use(express.json());
+app.use(cookieParser());
+
+app.use(
+  cors({
+    origin: process.env.CLIENT_HOST,
+    credentials: true,
+  }),
+);
+
+app.use('/auth', authRouter);
+app.use(errorMiddleware);
```

The application is missing an explicit handler for undefined routes. The requirements state that any route not defined should return a 404 error. You should add a catch-all middleware right before this error middleware to handle any requests that don't match `/auth` and generate a 404 error.